packages/beacon-state-transition/src/fast/util/flat.ts

```diff
@@ -1,13 +1,16 @@
-import {ObjectLike, readonlyEntries} from "@chainsafe/ssz";
+import {BLSPubkey, Bytes32, Validator} from "@chainsafe/lodestar-types/phase0";
```

Comment:
Can you import `phase0` and use it?
eg: `import {..., phase0} from "@chainsafe/lodestar-types";`